src/main/scala/loamstream/drm/Drmaa1Client.scala

```diff
@@ -75,10 +75,11 @@ final class Drmaa1Client(nativeSpecBuilder: NativeSpecBuilder) extends DrmaaClie
   /**
    * Kill the job with the specified id, if the job is running.
    */
-  override def killJob(jobId: String): Unit = {
-    debug(s"Killing Job '$jobId'")
+  override def killJob(taskId: DrmTaskId): Unit = {
+    debug(s"Killing Job '$taskId'")
```

Comment:
Feels weird to me that this is called `killJob` and the log is displaying `taskId` (as is the parameter). Is this intentional, maybe just a variable name not updated, or maybe something is not right here?

Reply:
The terminology is muddy.  Jobs and tasks are seperate things that are related and sometimes interchangeable.  LS only deals with tasks now, but that wasn't always (as explicitly) the case.  The Uger/GridEngine/DRMAA docs talk about this option in terms of "killing jobs", which is probably why this method got that name years ago.

That said, `killTask` would be more appropriate now.

Reply:
After a closer look, this method is just wrong: it will kill all the tasks in a task array, instead of just one, which is (I guess) the intent.

The upside is that this method is never called.  Confusing and unused = gone.